public/style.css

```
@@ -480,11 +488,32 @@ div, form, h3, h4, select, button {
   text-decoration: inherit;
 }
 
-.no-time-slot-warning {
+.no-time-slot-warning, .successful-booking-icon {
   position: relative;
   height: 24px;
   width: 24px;
   pointer-events: none;
   left: 0px;
   margin: 0px 10px 0px 0px;
-}
\ No newline at end of file
+}
+
+.sucessful-booking {
+  padding: .5rem;
+  background-color: #ffe1e8;
+  -ms-flex-negative: 0;
+  flex-shrink: 0;
+  max-width: 100%;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+.sucessful-booking-text-wrap {
```

Spelling error... quick fix. Also update the className in other parts of your code

```suggestion
.successful-booking-text-wrap {
```